src/pymovements/datasets/iitb_hgc.py

```diff
@@ -0,0 +1,159 @@
+from typing import Any
+
+from pymovements.dataset.dataset_definition import DatasetDefinition
+
+
+@dataclass
+class IITBHGC(DatasetDefinition):
+    """IITBHGC dataset :cite:p:`IITBHGC`.
+
+    This dataset includes monocular eye tracking data from 5 participants in a single
+    session. Recording both eyes at 2000 Hz, each participant read 500 generated paragraphs
+    and judges whether it was hallucinated or not.
+
+    Check the respective paper for details :cite:p:`IITBHGC`.
+
+    Attributes
+    ----------
+    name: str
+        The name of the dataset.
+
+    long_name: str
+        The entire name of the dataset.
+
+    has_files: dict[str, bool]
+        Indicate whether the dataset contains 'gaze', 'precomputed_events', and
+        'precomputed_reading_measures'.
+
+    resources: dict[str, list[dict[str, str]]]
+        A list of dataset gaze_resources. Each list entry must be a dictionary with the following
+        keys:
+        - `resource`: The url suffix of the resource. This will be concatenated with the mirror.
+        - `filename`: The filename under which the file is saved as.
+        - `md5`: The MD5 checksum of the respective file.
+
+    extract: dict[str, bool]
+        Decide whether to extract the data.
+
+    filename_format: dict[str, str]
+        Regular expression which will be matched before trying to load the file. Namedgroups will
+        appear in the `fileinfo` dataframe.
+
+    filename_format_schema_overrides: dict[str, dict[str, type]]
+        If named groups are present in the `filename_format`, this makes it possible to cast
+        specific named groups to a particular datatype.
+
+    trial_columns: list[str]
+            The name of the trial columns in the input data frame. If the list is empty or None,
+            the input data frame is assumed to contain only one trial. If the list is not empty,
+            the input data frame is assumed to contain multiple trials and the transformation
+            methods will be applied to each trial separately.
+
+    column_map: dict[str, str]
+        The keys are the columns to read, the values are the names to which they should be renamed.
+
+    custom_read_kwargs: dict[str, dict[str, Any]]
+        If specified, these keyword arguments will be passed to the file reading function.
+
+    Examples
+    --------
+    Initialize your :py:class:`~pymovements.dataset.Dataset` object with the
+    :py:class:`~pymovements.datasets.IITBHGC` definition:
+
+    >>> import pymovements as pm
+    >>>
+    >>> dataset = pm.Dataset("IITBHGC", path='data/IITBHGC')
+
+    Download the dataset resources:
+
+    >>> dataset.download()# doctest: +SKIP
+
+    Load the data into memory:
+
+    >>> dataset.load()# doctest: +SKIP
+    """
+
+    # pylint: disable=similarities
+    # The PublicDatasetDefinition child classes potentially share code chunks for definitions.
+
+    name: str = 'IITBHGC'
+
+    long_name: str = 'IITB-Hallucination Gaze corpus'
```

Comment:
as corpus is part of the name it should be capitalized

Reply:
done